js/functions.js

```diff
@@ -0,0 +1,37 @@
+function stringLengthCheck (str, maxlength) {
+  return str <= maxlength;
+}
+
+stringLengthCheck(25, 24);
+
+function isPalindrome (str) {
+  const firstStr = str.replaceAll(' ', '').toLowerCase();
+  let secondStr = '';
+
+  for (let i = firstStr.length - 1; i >= 0; i--) {
+    secondStr += firstStr.at(i);
+  }
+
+  return firstStr === secondStr;
+}
+
+isPalindrome('тОпот топоТ');
+
+function extractNumber (data) {
+  let result = '';
+  let str = data;
+
+  if (typeof(data) === 'number') {
+    str = data.toString();
```

Comment:
В принципе зачем функции делать ещё что-то? Можно сразу завершить
```suggestion
    return data
```